docs/datatypes/io.md

```diff
@@ -164,38 +164,30 @@ val composite = action.ensuring(cleanupAction)
 ### A full working example on using brackets
 ```scala mdoc:silent
 
-import scalaz.zio.{ App, DefaultRuntime, Task, UIO }
+import scalaz.zio.{App, Task, UIO}
 
-import java.io.{ File, FileInputStream }
+import java.io.{File, FileInputStream }
 import java.nio.charset.StandardCharsets
 
 object Main extends App {
 
-  val rt = new DefaultRuntime {}
-
   // run my bracket
   def run(args: List[String]) =
-    mybracket.orDie.map(_ => 0)
+    mybracket.orDie.const(-1) // return -1 on fail
```

Comment:
The `-1` would be returned on the success cases, so you want to return `0` here.